### PR TITLE
⬆️ 🥒 Upgrade cucumber 8

### PR DESCRIPTION
**Pull request by josemigallas**
`map_column!` and `map_header!` have been removed.

### CODE REVIEW

File: features/step_definitions/buyers_management/account_steps.rb [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
 Then /^(.*) for (?:buyer|provider|account) "([^"]*)"$/ do |action, org_name|
   account = Account.find_by!(org_name: org_name)
-  within('#' + dom_id(account)) { step action }
+  within("##{dom_id(account)}") { step action }
```
Comment (akostadinov) [inline features/step_definitions/buyers_management/account_steps.rb:34]: probably intentional duplicate symbol, but just double-checking

File: features/support/alerts.rb [hunk 2]
```diff
@@ -1,23 +1,18 @@
-# -*- coding: utf-8 -*-
-module Alerts
+# frozen_string_literal: true
 
-  def symbolize_headers(table)
-    table.map_headers! { |header| header.downcase.gsub(/\s+/, '_').to_sym }
-  end
```
Comment (akostadinov) [inline features/support/alerts.rb:6]: Although it has a single usage only I'd suggest moving it under `DataTableTransforms` and still use it in the step.

**Review by akostadinov — APPROVED** [review]
Just two small questions. Leaving it to your discretion to decide what to do with them.

File: features/support/alerts.rb (hunk 2 above)
Comment (josemigallas) [reply features/support/alerts.rb:6]: Move to DataTableTransforms.

**Pull request merged**